spec/requests/swagger_docs/v6/full_assessment_spec.rb

```diff
@@ -401,7 +401,7 @@
             assessment: { submission_date: "2022-06-06" },
             applicant: { date_of_birth: "2001-02-02", has_partner_opponent: false, receives_qualifying_benefit: false, employed: false },
             dependants: [
-              attributes_for(:dependant, relationship: "child_relative", in_full_time_education: true, monthly_income: 0, date_of_birth: "2015-02-11"),
+              attributes_for(:dependant, relationship: "child_relative", in_full_time_education: true, amount: 0, frequency: "monthly", date_of_birth: "2015-02-11").except(:amount, :frequency),
```

Comment:
It looks a bit strange adding the items into the hash and then taking them out again with the 'except'